prow/plugins/assign/assign.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package assign
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+const pluginName = "assign"
+
+var acceptRe = regexp.MustCompile(`(?mi)^\/(un)?assign(( [-\w]+?)*)\r?$`)
```

Comment:
I would recommend using `\w+(-\w+)?` for matching github handles, as it follows the "spec" more closely.

Also you may want to be more lenient on spaces before and after usernames?

Maybe also add an optional @

You can also make `( [-\w]+?)` non-capturable, I don't think you use that group explicitly.